colossalai/quantization/fp8.py

```diff
@@ -185,7 +187,7 @@ def all_reduce_fp8(
     return dist.all_reduce(tensor, op=op, group=group, async_op=async_op)
 
 
-@torch.compile(mode="max-autotune-no-cudagraphs", dynamic=False)
+@torch.compile(mode="max-autotune-no-cudagraphs", dynamic=False, disable=cuda_arch < 89)
```

Comment:
sm < 89 does not support fp8, not only torch compile with fp8

Reply:
Yes, but CI machine(3090, with sm<89) does support FP8 dtype(possibly by simulation) .So it can be used to verify the correctness of the fp8 collective algorithm.  But the torch.compile does not support the 3090 CI machine because of sm<89 so I disabled the compile.